app/src/main/res/layout/activity_benchmark_add_photo.xml

```diff
@@ -16,10 +16,10 @@
         android:layout_marginTop="156dp"
         android:layout_marginEnd="4dp"
         android:background="#00000000"
-        app:icon="@drawable/right"
+        app:icon="@drawable/right_array"
```

Comment:
Why are these arrows called "array"? Whats the connection with the array? Why are you assuming that they are used to iterate over array? Change it to "ic_right_arrow"